src/js/components/keyword/header/KeywordHeader.jsx

```diff
@@ -30,19 +30,25 @@ export class KeywordHeader extends React.Component {
             const primeCountUnits = MoneyFormatter.calculateUnitForSingleValue(primeCount);
             const primeAmountUnits = MoneyFormatter.calculateUnitForSingleValue(primeAmount);
 
-            let primeCountPrecision = 0;
-            if (primeCountUnits.unit !== 1) {
-                primeCountPrecision = 1;
+            if (primeCountUnits.unit >= 1000000) {
```

Comment:
To avoid future (or current) typos where we add/drop a zero by accident, can you use the `unitValues` in `moneyFormatter` for this?